advanced_java/enum/MyClass4.java

```diff
@@ -0,0 +1,29 @@
+public class MyClass4 {
+  public static void main(String[] args) {
+    Day myVar = Day.WEDNESDAY;
+
+    switch(myVar) {
+      case MONDAY:
+        System.out.println("Workday.");
+        break;
+      case TUESDAY:
+         System.out.println("Presentaion day.");
```

Comment:
Fix the typo.

Reply:
where ?

Reply:
It should be "Presentation day." not "Presentaion day."